stories/components/card/card.handlebars

```diff
@@ -1,13 +1,27 @@
-<ul class="card-list list--unstyled">
-  <li class="card">
-    <div class="card__body">
-      <a class="card__title" href="{{linkTarget}}">{{titleText}}</a>
-      <p class="card__body-text">{{bodyText}}</p>
+{{#if link}}
+  <ul class="card-list list--unstyled">
```

Comment:
I'm just noticing this now, but can we incorporate the `list--unstyled` rules into the `card-list` rules so we don't have to add two rules here?